src/main/java/org/cyclops/evilcraftcompat/modcompat/forestry/FarmableUndeadSapling.java

```diff
@@ -0,0 +1,87 @@
+package org.cyclops.evilcraftcompat.modcompat.forestry;
+
+import forestry.farming.logic.CropDestroy;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.Blocks;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import forestry.api.farming.ICrop;
+import forestry.api.farming.IFarmable;
+import forestry.core.network.packets.PacketFXSignal;
+import forestry.core.utils.NetworkUtil;
+import org.cyclops.evilcraft.block.UndeadSaplingConfig;
+import org.cyclops.evilcraft.item.HardenedBloodShardConfig;
+
+public class FarmableUndeadSapling implements IFarmable {
```

Comment:
Why is this class needed? Can't you just use Forestry's `FarmableSapling` class directly?

Reply:
No, I can't. You're build script uses an old version of forestry, which does not implement FarmableSapling yet.

Reply:
Perhaps it's a good time to update then, as the `add-farmable-sapling` message is broken anyways.